File: app/src/main/java/com/example/ign_app/model/data/Authors.java
package com.example.ign_app.model.data;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Authors {

    @SerializedName("name")
    @Expose
    private String name;

    @SerializedName("thumbnail")
    @Expose
    private String thumbnail;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getThumbnail() {
        return thumbnail;
    }

    public void setThumbnail(String thumbnail) {
        this.thumbnail = thumbnail;
    }

    @Override
    public String toString() {
        return "Authors{" +
                "name='" + name + '\'' +
                ", thumbnail='" + thumbnail + '\'' +
                '}';
    }
}

File: app/src/main/java/com/example/ign_app/articlemodel/data/Authors.kt
package com.example.ign_app.articlemodel.data;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

 class Authors {

    @SerializedName("name")
    @Expose
    var name: String? = null

    @SerializedName("thumbnail")
    @Expose
     var thumbnail:String? = null

    override fun toString():String {
        return "Authors{" +
                "name='" + name + '\'' +
                ", thumbnail='" + thumbnail + '\'' +
                '}';
    }
}
